src/odemis/acq/path.py

```diff
@@ -288,7 +290,20 @@
                  'light-aligner': {'x': "MD:" + model.MD_FAV_POS_ACTIVE,
                                    #'z': "MD:" + model.MD_FAV_POS_ACTIVE
                                    },
-                 }),
+                }),
+            'tunnel-lens-align': ("sp-ccd",
+                {'chamber-light': {'power': 'off'},
+                 'spec-selector': {'x': "MD:" + model.MD_FAV_POS_ACTIVE},
```

Comment:
Where exactly does it need to be 'double checked'? Cannot seem to find a spot where to do that in path.py or should that be done in sparc2_align_tab.py where the selectorsToPath() is called?

Reply:
I meant that it should be removed, because it's already done automatically. However, after removing that line, you should run the simulator and confirm that indeed the spec-selector does go to the right position when selecting this optical mode (tunnel-lens-align).